### ISSUE TITLE
missing method exception

### ISSUE THREAD

**Issue by babapro45** [RocketMod/Rocket.Unturned#89]
can you look at that please https://imgur.com/gallery/sGWEHSm or https://imgur.com/a/sGWEHSm

**Comment by Game4Freak** [comment]
Its a problem of unturned itself. The last update added it

**Comment by babapro45** [comment]
how can i fix? do you know?

**Comment by babapro45** [comment]
and in my server  when i /P add (name) (Job) the job(tag) is not showing and i tried to exit and join again
but now showing tag is it about this error or anything different can help to fix??

**Comment by Game4Freak** [comment]
TermsGames mentioned this on discord:
Change kickinsteadreject to true in your globalban config.

But you can ignore that error it shouldnt affect the functionality of the game

**Comment by Game4Freak** [comment]
the /p thing is not part of the bug the bug is a unturned only error wich has nothing to do with rocketmod

**Comment by babapro45** [comment]
can you explain more i dont have good english and i opened my server on short folder

**Comment by Game4Freak** [comment]
join the rocketmod discord and ask for fixes there:

https://discordapp.com/invite/N8yXrAx

**Comment by persiafighter** [comment]
Issue 1 (Missing Method):
You cannot fix it. The code has to be changed in the legacy branch to use the latest reference of unturned and use whatever is equivalent to SDG.Unturned.Provider.reject (no clue if there will be anything equivalent)

Issue 2 (Prefix/Suffix not showing):
You cannot fix it. The code again has to be changed in the legacy branch, to modify whatever has become the new CharacterName field, ie: whatever now is the name being displayed in chat and join/leave messages.
(I'm assuming that this is why it's happening, but I may be wrong and it might be happening because of Issue 1, which if that's the case, then issue 1 should get fixed and then this will solve itself)

In conclusion:
You can't fix it. It has to be fixed code wise.

**Comment by babapro45** [comment]
hope authorities will fix

**Comment by rube200** [comment]
I'm not sure but I think I went around the problem, I used something like oncheck(evento)

**Comment by Trojaner** [comment]
RocketMod 4 will not receive any updates
